conda.yaml

```diff
@@ -0,0 +1,10 @@
+name: yiqing-test
+channels:
+  - defaults
+dependencies:
+  - numpy>=1.14.3
```

Comment:
I could recommend mentioning the minimum version of Python required in the README .

Reply:
The conda configuration here is to show that the mlflow run without an issue when we configure the project using conda_env. For our use case with docker_env, this should have been specified by the docker container.